src/test/java/com/arpnetworking/logback/serialization/steno/StenoSerializationHelperTest.java

```diff
@@ -32,19 +35,30 @@
  */
 public class StenoSerializationHelperTest {
 
-    @Before
-    public void setUp() {
-        _objectMapper = new ObjectMapper();
-        _encoder = new StenoEncoder();
-        _objectMapper.setAnnotationIntrospector(new StenoAnnotationIntrospector(_objectMapper));
+    @Test
+    public void testNonThrowableProxy() throws IOException {
+        final ObjectMapper objectMapper = new ObjectMapper();
+        final StringWriter jsonWriter = new StringWriter();
+        final JsonGenerator jsonGenerator = objectMapper.getFactory().createGenerator(jsonWriter);
+        jsonGenerator.writeStartObject();
+        objectMapper.setAnnotationIntrospector(new StenoAnnotationIntrospector(objectMapper));
+        StenoSerializationHelper.serializeThrowable(
+                new AlternateIThrowableProxy(),
+                jsonGenerator,
+                objectMapper);
+        jsonGenerator.writeEndObject();
+        jsonGenerator.close();
+        Assert.assertEquals(
```

Comment:
This particular unit test _only_ targets the instanceof statement in StenoSerializationHelper by providing an instance of IThrowableProxy which is not a ThrowableProxy (hence the name testNonThrowableProxy). 

There are already unit tests in StenoEncoderTest around the standard exception field encoding; for example:

* testEncodeArrayWithException
* testEncodeArrayWithCausedException
* testEncodeArrayWithSuppressedException
* testEncodeArrayWithNullSuppressedException

Those unit tests cannot test the instanceof branch because they accept a Throwable and our code does not control the wrapping of it in an IThrowableProxy.

If there is a case missing here, please elaborate.